src/api/models.py

```diff
@@ -1,3 +1,26 @@
 from django.db import models
 
 # Create your models here.
+class Risk_Factors(models.Models):
+    risk_factor_id = models.IntegerField()
```

Comment:
If this is the primary key for this model, it should be marked as so.
[https://docs.djangoproject.com/en/2.2/ref/models/fields/#django.db.models.Field.primary_key]